doc/install_mne_python.rst

```diff
@@ -2,101 +2,112 @@
+    changed on the command line with the ``-n`` flag; see ``conda env create
+    --help`` for more info.
+
+Make sure you activate the environment (``conda activate mne``) each
+time you open a terminal, or put the activation command in your ``.bashrc``
+or ``.profile`` so that it happens automatically.
 
 .. admonition:: |apple| macOS
   :class: note
 
-  If you are on macOS, you need to manually update PyQt5. This step is not needed on Linux, and even breaks things on Windows.
+  If you are on macOS, you need to manually update PyQt5. This step is not
+  needed on Linux, and even breaks things on Windows.
 
   .. code-block:: console
 
     $ pip install --upgrade "pyqt5>=5.10"
 
+Testing your installation
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-3. Check that everything works
-##############################
-
-To make sure everything installed correctly, type the following command in a terminal:
+To make sure MNE-python installed correctly, type the following command in a
+terminal:
 
 .. code-block:: console
 
-    $ python
+    $ python -c 'import mne; mne.sys_info()'
 
-This should open an interactive Python prompt, where you can type::
+This should display some system information along with the versions of
+MNE-python and its dependencies. Typical output looks like this:
 
-    >>> import mne
-
-If you get a new prompt with no error messages, you should be good to go!
-
-.. admonition:: |windows| Windows
-  :class: note
-
-  If you are on Windows, you might have to use the IPython magic command ``%gui qt``
-  after importing MNE, Mayavi or PySurfer (see `here <https://github.com/ipython/ipython/issues/10384>`_):
-
-  .. code-block:: ipython
-
-     In [1]: from mayavi import mlab
-     In [2]: %gui qt
-
-The ``$ conda env create ...`` step sometimes emits warnings, but you can ensure
-all default dependencies are installed by listing their versions with::
+.. code-block:: console
 
-    >>> mne.sys_info()  # doctest:+SKIP
-    Platform:      Linux-4.18.0-13-generic-x86_64-with-debian-buster-sid
-    Python:        3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34)  [GCC 7.3.0]
-    Executable:    /home/travis/miniconda/envs/test/bin/python
-    CPU:           x86_64: 48 cores
-    Memory:        62.7 GB
+    Platform:      Linux-4.15.0-44-generic-x86_64-with-debian-buster-sid
+    Python:        3.6.7 |Anaconda, Inc.| (default, Oct 23 2018, 19:16:44)  [GCC 7.3.0]
+    Executable:    /opt/miniconda3/envs/mne/bin/python
+    CPU:           x86_64: 8 cores
+    Memory:        7.6 GB
 
-    mne:           0.17.0
+    mne:           0.18.dev0
     numpy:         1.15.4 {blas=mkl_rt, lapack=mkl_rt}
-    scipy:         1.2.0
+    scipy:         1.1.0
     matplotlib:    3.0.2 {backend=Qt5Agg}
 
     sklearn:       0.20.2
-    nibabel:       2.3.3
-    mayavi:        4.7.0.dev0 {qt_api=pyqt5, PyQt5=5.10.1}
+    nibabel:       2.3.1
+    mayavi:        4.7.0.dev0 {qt_api=pyqt5, PyQt5=5.9.2}
     cupy:          Not found
-    pandas:        0.24.0
```

Comment:
please revert these changes to the output, I had it (recently) set to what you'd get with the latest MNE env.